main/apogee.h

```diff
@@ -6,8 +6,8 @@
 #include "Ewma.h"
 #include "utils.h"
 
-#define G_FORCE_TO_LAUNCH 3 //if acceleration exceeds this number the rocket will assume it has been launched
-#define MAX_APOGEE_ACCEL 2 //we can rule out apogee if acceleration is about this amount (Gs)
+static constexpr float G_FORCE_TO_LAUNCH = 3 //if acceleration exceeds this number the rocket will assume it has been launched
```

Comment:
since this is being compared to `double`s it should be a `double` as well instead of a `float`